source/OutfitterPanel.cpp

```diff
@@ -74,6 +87,19 @@ namespace {
 		}
 		return toRefill;
 	}
+
+
+
+	static bool IsLicense(const string &name)
```

Comment:
functions in the unnamed namespace are implicitly static
```suggestion
	bool IsLicense(const string &name)
```